amundsen_application/static/js/components/TableDetail/index.tsx

```diff
@@ -158,7 +158,7 @@ class TableDetail extends React.Component<TableDetailProps & RouteComponentProps
                   editable={ data.is_editable }
                 />
               </EditableSection>
-              { !data.table_description && notificationsEnabled() && <RequestDescriptionText/> }
```

Comment:
It was an intentional decision to have the `RequestDescriptionText` option even when there is a description. The decision considered the case of an existing description not being clear enough or needing more detail, and this would allow users a pathway to request an improved description.

Reply:
Sounds good, I agree that in such a case
> an existing description not being clear enough or needing more detail

users should be able to request changes. That's why I removed `!data.table_description` part of the statement - it makes "Request Description" link hidden

Reply:
Ah my mistake, for some reason my brain processed this logic backwards.